components/coachindicator/index.css

```diff
@@ -0,0 +1,99 @@
+/*!
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import "animation.css";
+
+.spectrum-CoachIndicator {
+  --spectrum-coach-indicator-ring-border-size: var(--spectrum-border-width-200);
+  --spectrum-coach-indicator-min-inline-size: calc(var(--spectrum-coach-indicator-ring-diameter) * 3);
+  --spectrum-coach-indicator-min-block-size: calc(var(--spectrum-coach-indicator-ring-diameter) * 3);
+  --spectrum-coach-indicator-ring-inline-size: var(--spectrum-coach-indicator-ring-diameter);
+  --spectrum-coach-indicator-ring-block-size: var(--spectrum-coach-indicator-ring-diameter);
+
+  --spectrum-coach-indicator-top: calc(var(--spectrum-coach-indicator-ring-diameter) * 0.75);
+  --spectrum-coach-indicator-left: calc(var(--spectrum-coach-indicator-ring-diameter) * 0.75);
+
+  /* animation */
+  --spectrum-coach-animation-indicator-ring-duration: 3000ms;
```

Comment:
This should be encapsulated inside this component. This should be a part of the system. This will not allow us to override in reduced motion. Can this be done without us overrding with --mod property?

Reply:
> This should be encapsulated inside this component.

Should _not_ be, right? Just want to be sure that's what you mean.

Yeah, I think this is probably one we should move out and into the custom `/tokens/` package, similar to the [ones we already have](https://github.com/adobe/spectrum-css/blob/814c49e22098f22044893c64f21744aa62cee8f6/components/tokens/custom-spectrum/custom-vars.css#L19-L36) in there. Interesting that this value is set to `3000ms`, and we don't have just below and just above that already defined in the custom tokens. We copied those values over from the older vars system.

@jenndiaz what do you think about abstracting this out and placing it into the custom tokens? The naming might be a little interesting given what the current names are.

Reply:
@Rajdeepc I changed --spectrum-coach-animation-indicator-ring-duration: 3000ms; to use a custom animation token, does this solve the issue you were having with overide and reduced motion?

Reply:
great, I have a separate PR up to add the custom token: https://github.com/adobe/spectrum-css/pull/2174

Reply:
will there be another release of `"@spectrum-css/coachindicator"` for this change?
Now its on  `"@spectrum-css/coachindicator": "^1.0.0-alpha.1" `

Reply:
@Rajdeepc yes! Was released earlier today https://github.com/adobe/spectrum-css/pull/1992#issuecomment-1726068766